iib/workers/tasks/build_merge_index_image.py

```diff
@@ -440,43 +440,73 @@ def is_bundle_version_valid(
     v4.7   | included  | NOT included |   included   |  included   |  included
     -------------------------------------------------------------------------------
     v4.8   | included  | NOT included | NOT included |  included   |  included
+
+
+           |  "=v4.5|=v4.6"  |   "=v4.5|>=v4.7"   |   "<=v4.7"   |  "<=v4.6|>=v4.8"
```

Comment:
I don't think `"<=v4.6|>=v4.8"` is a supported version in Pyxis.

Reply:
should I remove it?

Reply:
Yeah, instead =v4.6|=v4.7|>=v4.8 (when v4.8 is not deprecated) should be there, no?

Reply:
@yashvardhannanavati please check